openshift/ci-operator/build-image/Dockerfile

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 
 # Dockerfile to bootstrap build and test in openshift-ci
-FROM registry.ci.openshift.org/openshift/release:golang-1.17
+FROM registry.ci.openshift.org/openshift/release:golang-1.18
 
 # Add kubernetes repository
 ADD openshift/ci-operator/build-image/kubernetes.repo /etc/yum.repos.d/
 
-RUN yum install -y kubectl ansible httpd-tools
+RUN yum install -y kubectl httpd-tools
 
-RUN GO111MODULE=on go get github.com/mikefarah/yq/v3
+RUN wget -q https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq &&\
+    chmod +x /usr/bin/yq
```

Comment:
@pierDipi fyi we'll probably have to take a look at upgrading `yq` to v4. Rather sooner than later?

Reply:
why? to avoid installing the specific version we need?